templates/back/signalement/view/edit-modals/edit-coordonnees-tiers.html.twig

```diff
@@ -47,8 +47,7 @@
                             <div class="fr-select-group">
                                 <label class="fr-label" for="lien">Lien avec l'occupant</label>
                                 <select class="fr-select" name="lien">
-                                    <option value="" disabled hidden>Sélectionnez un choix</option>
-                                    <option value="" {{ signalement.lienDeclarantOccupant is same as '' ? 'selected' : '' }}>N/C</option>
+                                    <option value="" {{ signalement.lienDeclarantOccupant not in ['PROCHE', 'VOISIN', 'SECOURS', 'BAILLEUR', 'PRO', 'AUTRE'] ? 'selected' : '' }}></option>

```

Comment:
Il me semble qu'il y a un Enum pour le lienDeclarantOccupant. Ce ne serait pas plus safe de se référer à l'enum dans toute cette partie ?

Reply:
En effet il y'a un Enum, par contre je n'ai pas su comment l'utiliser ici donc j'ai laissé en l'état.

Reply:
Dans l'enum OccupantLink, il faudrait ajouter une fonction de ce type

`
    public static function names(): array
    {
        return array_column(self::cases(), 'name');
    }
`
 Puis dans le twig tu peux utiliser l'enum comme ça : 
 
 `
{% set OccupantLink = enum('\\App\\Entity\\Enum\\OccupantLink') %}
{{ dump(OccupantLink.names()) }}

{% if 'PROCHE' in OccupantLink.names()  %}
    fkdslmfkdslmkfdsmlk
{% endif %}
`

Reply:
Donc en fait comme ça je pense 

```
set OccupantLink = enum('\\App\\Entity\\Enum\\OccupantLink') 
<option value="" {{ signalement.lienDeclarantOccupant not in OccupantLink.names() ? 'selected' : '' }}></option>
```

Reply:
C'est fait, merci